icon/Icon.js

```diff
@@ -238,6 +246,7 @@ import {
     faSearch as faSearchSolid,
     faSignIn as faSignInSolid,
     faSignOut as faSignOutSolid,
+    faSkull as faSkullSolid,
     faSun as faSunSolid,
```

Comment:
Where are we using this?   :-)

Reply:
In Toolbox > Toolbars.  Gotta have at least one skull someplace. 💀Maybe we can add it to some "fatal error" dialog as well to get our money's worth.